app/src/main/java/com/duckduckgo/app/browser/di/BrowserModule.kt

```diff
@@ -202,7 +203,16 @@ class BrowserModule {
     }
 
     @Provides
-    fun webViewDatabaseLocator(context: Context): WebViewDatabaseLocator = WebViewDatabaseLocator(context)
+    @Named("webViewDbLocator")
+    fun webViewDatabaseLocator(context: Context): DatabaseLocator = WebViewDatabaseLocator(context)
+
+    @Provides
+    @Named("authDbLocator")
+    fun authDatabaseLocator(context: Context): DatabaseLocator = AuthDatabaseLocator(context)
+
+    @Provides
+    @Named("authDbCleaner")
```

Comment:
I think we should remove this named qualifier. There is only one implementation of the `DatabaseCleaner`

Reply:
Done!